src/jwplayer/players/shared/JwPlayerWrapperWithStrategyRules.tsx

```diff
@@ -41,6 +42,8 @@ const JwPlayerWrapperWithStrategyRules: React.FC<JwPlayerWrapperProps> = ({
 	const strategyRulesPlacementId = 'KmMLkvao';
 	const recommendationPlaylistId = 'FOhaD53w';
 	const communicationService = getCommunicationService();
+	const prerollAdTag = vastUrl ?? '';
```

Comment:
Using multiple names for the same thing can be confusing. We already have "vastUrl" and "adTagUrl" as different names for the VAST ad tag URL value. Let's avoid making things even more complicated and at least always include the "url" part 😉 

```suggestion
	const prerollAdTagUrl = vastUrl ?? '';
```

Reply:
`preroll_ad_tag` is the name of JWP custom parameter (`jwDataStore.custom[strategyRulesPlacementId].preroll_ad_tag`) so I'd leave it as is to be consistent. Also `jwplayer-fandom` lives on borrowed time so I wouldn't care that much about naming ;)